vendor/modules.txt

```diff
@@ -230,6 +230,9 @@ github.com/exponent-io/jsonpath
 github.com/fatih/color
 # github.com/fsnotify/fsnotify v1.4.9
 github.com/fsnotify/fsnotify
+# github.com/gardener/etcd-druid/api v0.6.0
```

Comment:
Can you use an up-to-date version?

Reply:
Updated to use `v0.7.3` in [`eda8f83`](https://github.com/gardener/etcd-backup-restore/pull/403/commits/eda8f831bd047009f71581346da75ef3e4d4fbd0)